mcodingbot/tasks/stats.py

```diff
@@ -122,27 +121,32 @@ async def get_stats(bot: Bot) -> Stats:
     return _last_known_stats
 
 
+def format_float(n: int | float) -> int | float:
+    if float(n).is_integer():
+        return int(n)
+    return n
+
+
+def floor(number: int | float, ndigits: int = 0) -> float:
+    n: int | float = 10**ndigits
+    return int(number * n) / n
+
+
 def display_stats(stat: int | float) -> str:
-    if stat < 10**3:
+    if stat < 1_000:
```

Comment:
Imo 10**N was more readable here

Reply:
I guess that's subjective; also `1e3` is another option :trollface:

Reply:
I think the relationship between 10**N and 10**N-3 shows the intent more clearly than a different number of 0s